cumulusci/cli/org.py

```diff
@@ -240,32 +235,56 @@ def org_default(runtime, org_name, unset):
             click.echo("There is no default org")
 
 
-@org.command(name="import", help="Import a scratch org from Salesforce DX")
+@org.command(name="import", help="Import an org from Salesforce DX")
 @click.argument("username_or_alias")
 @orgname_option_or_argument(required=True)
 @pass_runtime(require_keychain=True)
-def org_import(runtime, username_or_alias, org_name):
-    org_config = {"username": username_or_alias}
-    scratch_org_config = ScratchOrgConfig(
-        org_config, org_name, runtime.keychain, global_org=False
+def org_import(runtime: CliRuntime, username_or_alias: str, org_name: str):
+    # Import the org from the SFDX keychain as an SfdxOrgConfig
+    # The `sfdx` key ensures we can reload using the right class.
+    org_config = SfdxOrgConfig(
+        {"username": username_or_alias, "sfdx": True},
+        org_name,
+        runtime.keychain,
+        global_org=False,
     )
-    scratch_org_config.config["created"] = True
 
-    info = scratch_org_config.sfdx_info
-    if not info.get("created_date"):
-        raise click.UsageError(
-            "cci org import only works for locally created "
-            "scratch orgs.\nUse `cci org connect` for other orgs."
+    # Determine if we received a locally-created scratch org
+    # or some other org (which we'll treat as persistent)
+
+    info = org_config.sfdx_info
+    if info.get("created_date"):
```

Comment:
I paused over this enough times that I'm thinking something like the following may be better. The real question is, does that method make sense as part of `SfdxOrgConfig` 🤔 . At least abstracting the logic into a method would allow for a docstring about how/why we determine this the way we do. I'm probably overthinking it at this point 😂 .
```suggestion
    is_scratch = org_config.is_scratch_org()
    if is_scratch:
```

Reply:
I shied away from making changes to the method set or properties of the OrgConfig because the existing code is _so_ nasty with the combination of methods, properties, and config values - some of which override each other. I want to tear it all out and refactor it, but not in this PR!